### PR TITLE
feat(web): add optional hideable scrollbar to scrollcontrols

### LINKED ISSUE #637: Add an option to hide the scrollbar in Scroll Controls
### It would be great if we can have an option maybe something like a prop that can Hide the scrollbar without losing functionality in ScrollControls

Doesn't seem like there could be any drawbacks on this, but maybe someone can suggest?

I managed to do it by changing the file ScrollControls.js  at line 76

>    el.style.top = '0px';
    el.style.left = '15px'; // changed it here, to hide the scroll bar and retain functionality
    fixed.style.position = 'sticky';

### PR DESCRIPTION
**Pull request by CaptEmulation**
### Why

Fixes #637

### What

Sure, one could just know the right style to apply here in order to hide the scrollbar (I do it myself) but this gives users a nice option.

### Checklist

- [x] Documentation updated
- [x] Ready to be merged

### CODE REVIEW

File: src/web/ScrollControls.tsx [hunk 1]
```diff
@@ -108,7 +111,7 @@ export function ScrollControls({
     el.style[horizontal ? 'overflowX' : 'overflowY'] = 'auto'
     el.style[horizontal ? 'overflowY' : 'overflowX'] = 'hidden'
     el.style.top = '0px'
-    el.style.left = '0px'
+    el.style.left = hideScrollbar ? '15px' : '0px'
```
Comment (drcmda) [inline src/web/ScrollControls.tsx:114]: shouldn't it be reversed?

**Review by drcmda — CHANGES_REQUESTED** [review]

File: src/web/ScrollControls.tsx (hunk 1 above)
Comment (CaptEmulation) [reply src/web/ScrollControls.tsx:114]: really sorry but been busy with some unexpected work. Going to get back to you as soon as I can with an example, but may be another couple of weekends

**Comment by CaptEmulation** [comment]
going ahead and closing since I currently have no need for this change

**Comment by rickyzhangca** [comment]
need this for the immersive ux without seeing scrollbar jumping around, especially in infinite scrolls...

**Pull request closed without merging**